common/clear_history.sh

```diff
@@ -65,6 +65,7 @@ rm -rf /tmp/nvidia* /tmp/MLNX* /tmp/ofed.conf /tmp/dkms* /tmp/*mlnx*
 rm -rf /run/cloud-init
 rm -rf /root/*
 rm -rf /usr/tmp/dnf*
+rm -f /opt/azurehpc/test/azurehpc-health-checks/health.log
```

Comment:
Would this break the build if NHC is not run during customization?

Reply:
NHC is always run in our pipelines.

I'll have to double check but I thought removing something that wasn't there was a no-op

Reply:
Ah, my point is that customers that use these scripts might not use NHC in their build workflows. Maybe we can keep this commented and uncomment in our build pipeline (similar to some of the other cleanup/deprovision code)?